website/docs/r/pi_volumes_attach.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM : ibm_pi_volumes_attach"
+description: |-
+  Manages volumes in Power Virtual Server cloud.
+---
+
+# ibm_pi_volumes_attach
+
+Attach volumes to a Power Systems Virtual Server instance, see [volumes](https://cloud.ibm.com/apidocs/power-cloud#pcloud-v2-pvminstances-volumes-post)
+
+## Example Usage
+
+The following example attach volumes to power virtual instance.
+
+```terraform
+resource "ibm_pi_volumes_attach" "pi_volumes_attach_instance" {
+    pi_cloud_instance_id = "a1a23e24-220d-4a20-a678-c8c5e84056bd"
+    pi_instance_id       = "49c66fb5-30af-4d94-ab2f-3ec1bf16e58c"
+    pi_volume_ids        = ["600850d6-4b38-40cf-857d-5e84147bdf89","d783bb14-efeb-4ded-9847-75a53481b60c", "ce0ab50a-9fd2-4c61-a609-46a9e7cc833d", "a09feb5b-e222-4f30-974b-ef47bb327a6e", "23679db9-c373-47f2-8906-8cb383c089b9"]
+}
+```
+
+## Note
+
+* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+* If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  * `region` - `lon`
+  * `zone` - `lon04`
+
+  Example usage:
+  
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+## Timeouts
+
+pi_volumes_attach provides the following [Timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) configuration options:
+
+* `create` - (Default 10 minutes) Used for attaching volumes to a power virtual server instance.
+
+## Argument Reference
+
+You can specify the following arguments for this resource.
+
+* `pi_boot_volume_id` - (Optional, Forces new resource, String) Primary Boot Volume Id.
+* `pi_cloud_instance_id` - (Required, Forces new resource, String) Cloud Instance ID of a PCloud Instance.
+* `pi_instance_id` - (Required, Forces new resource, String) PCloud PVM Instance ID.
+* `pi_volume_ids` - (Required, Forces new resource, List) List of volumes to be attached to a PVM instance.
+
+## Attribute Reference
+
+After your resource is created, you can read values from the listed arguments and the following attributes.
+
+* `id` - The unique identifier of volumes attach. The ID is composed of <pi_cloud_instance_id>/<ppi_instance_id>
```

Comment:
Something is not stylized properly from the docpreview's point of view:
![Screenshot 2024-03-05 at 8 39 03 AM](https://github.com/powervs-ibm/terraform-provider-ibm/assets/90468712/451421bc-54eb-4ff8-a3e6-7d8628a8743e)

Reply:
oh I think it should be:
```
`<pi_cloud_instance_id>/<ppi_instance_id>`
```

Reply:
Done